docs/vsphere_ci/README.md

```diff
@@ -43,13 +43,14 @@ associated with this public key is what will be used by WMCO to configure VMs cr
 deploying WMCO, this private key will be provided by the user in the form of a Secret.
 
 The [autounattend.xml](scripts/autounattend.xml) file must be edited to update the value of 
-`WindowsPassword` with a user provided password. Then, it executes the following steps:
+`WindowsPassword` with a user provided password. The `ProductKey` must also be updated with a proper value.
+Then, it executes the following steps:
```

Comment:
Since this line is being changed anyway, please reword this to be more correct:

autounattend.xml specifies that the following steps should occur: